u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java

```diff
@@ -97,6 +112,7 @@ public SecurityKeyData processRegistrationResponse(RegistrationResponse registra
     String browserDataBase64 = registrationResponse.getBd();
     String rawRegistrationDataBase64 = registrationResponse.getRegistrationData();
 
+    Log.info(">> rawRegistrationDataBase64: " + rawRegistrationDataBase64);
```

Comment:
What's the difference between the data logged here and in line 130 (other than the encoding)?